ts/calculator.ts

```diff
@@ -1,66 +1,66 @@
-import { POWER, calculator_buttons } from './buttons.js';
-
-
-const trig_container = document.getElementById("trigo_funcs")!;
-const trig_block = document.getElementById("trig-block")!;
-const func_container = document.getElementById("funcs")!;
-const f_block = document.getElementById("f-block")!;
-const func_arrow = document.getElementById("func_arrow")!;
-const trig_arrow = document.getElementById("trig_arrow")!;
-const other_funcs = document.getElementById("other-funcs")!;
-
-other_funcs.addEventListener("click", () => {
+import { POWER, calculatorButtons } from './buttons.js';
+
+
+const trigContainer = document.getElementById("trigo_funcs")!;
+const trigBlock = document.getElementById("trig-block")!;
+const funcContainer = document.getElementById("funcs")!;
+const fBlock = document.getElementById("f-block")!;
+const funcArrow = document.getElementById("func_arrow")!;
+const trigArrow = document.getElementById("trig_arrow")!;
+const otherFuncs = document.getElementById("other-funcs")!;
+type Expression = string | number | boolean;
+otherFuncs.addEventListener("click", () => {
     changeFuncs();
 })
-const FACTORIAL = "FACTORIAL";
-const OPERATORS = ["+", "-", "*", "/"];
-interface Exp_data {
-    operation: (string | number | boolean)[],
-    formula: (string | number | boolean)[],
+const factorialSearch = "FACTORIAL";
+const operators = ["+", "-", "*", "/"];
+interface ExpData {
+    operation: (Expression)[],
```

Comment:
You can remove parenthesis. It is not required if you create a custom type alias.
Alternatively,  use can also write `Array<Expression>`. It has same meaning as `Expression[]`